service/storage_proxy.cc

```diff
@@ -4951,7 +4951,7 @@ class data_read_resolver : public abstract_read_resolver {
             for (const version& ver : v) {
                 if (ver.par) {
                     mutation_application_stats app_stats;
-                    m.partition().apply(*schema, ver.par->mut().partition(), *schema, app_stats);
+                    co_await apply_gently(m.partition(), *schema, ver.par->mut().partition(), *schema, app_stats);
                     co_await coroutine::maybe_yield();
```

Comment:
@denesb 
The call to maybe_yield is redundant now with this change.

Reply:
https://github.com/scylladb/scylladb/pull/21984